src/experiments/rl_envs/JuliaRL_SAC_Pendulum.jl

```diff
@@ -26,12 +26,13 @@ function RLCore.Experiment(
     init = glorot_uniform(rng)
 
     create_policy_net() = NeuralNetworkApproximator(
-        model = SACPolicyNetwork(
-            Chain(Dense(ns, 30, relu), Dense(30, 30, relu)),
-            Chain(Dense(30, 1, initW = init)),
-            Chain(
-                Dense(30, 1, x -> clamp(x, typeof(x)(-2), typeof(x)(2)), initW = init),
+        model = GaussianNetwork(
+            pre = Chain(
+                Dense(ns, 30, relu), 
+                Dense(30, 30, relu),
             ),
+            μ = Chain(Dense(30, 1, initW = init)),
+            logσ = Chain(Dense(30, 1, x -> clamp.(x, typeof(x)(-10), typeof(x)(2)), initW = init)),
```

Comment:
So why is the lower bound changed to -10 here?

Reply:
Originally it was -20 after which I changed it to -2 in a previous PR when I mistook this output to be the action and not the standard deviation. I now changed it back to something smaller to allow the action distribution to be very precise if that is what it wants. But I felt -20 was a bit over the top, and it is what made me confused in the beginning since we had action space -2..2 and this clip was -20..2 so it looked like a typo. This might not happen as easily now when we have named the parts of the network, but still felt -20 was more than needed for the example so I went with -10 instead.